packages/react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -57,7 +57,6 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
         <div
           className={this._classNames.content}
           id={id}
-          role="tooltip"
```

Comment:
Ah, this was because a live region that contains `role="tooltip"` has support issues with text content calculation in a couple screen readers (i.e. will not read the text in the live region). Since `role="tooltip"` provides literally no benefit (no harm either TBF), I removed it :D